include/zephyr/drivers/fuel_gauge.h

```diff
@@ -37,61 +38,70 @@ extern "C" {
 #define FUEL_GAUGE_AVG_CURRENT 0
 
 /** Battery current (uA); negative=discharging */
-#define FUEL_GAUGE_CURRENT		    FUEL_GAUGE_AVG_CURRENT + 1
+#define FUEL_GAUGE_CURRENT                  FUEL_GAUGE_AVG_CURRENT + 1
```

Comment:
Thanks again for running `clang-format`. Can we separate the `clang-format` changes into another commit? Other reviewers will probably want the same.